sigstore/_cli.py

```diff
@@ -1222,3 +1232,16 @@ def _fix_bundle(args: argparse.Namespace) -> None:
         args.bundle.write_text(bundle.to_json())
     else:
         print(bundle.to_json())
+
+
+def _update_trust_root(args: argparse.Namespace) -> None:
+    # Simply creating the TrustedRoot in online mode is enough to perform
+    # a metadata update.
+    if args.staging:
+        trusted_root = TrustedRoot.staging(offline=False)
+    else:
+        trusted_root = TrustedRoot.production(offline=False)
+
+    _console.print(
+        f"Trust root updated: {len(trusted_root.get_fulcio_certs())} Fulcio certificates"
+    )
```

Comment:
It's not the best/most useful output, but it should be a visually obvious sign that the step hasn't quietly failed.

Another option here is to dump the trust root as JSON. Not sure how I feel about that, since I don't want people to assume a stable interface on this subcommand quite yet 😅

Reply:
I think that would be excessive verbose and/or might confuse people as to whether it was written to the file or whether the user needs to redirect the output. Perhaps it could print cache path too, like `written to ...`.

Reply:
I'm maybe not following which file you are talking about but I think exposing  `trusted_root.json` location is the only thing that makes sense (as users can't do much with TUF files): that is easily available in all TrustedRoot constructors so you could store the path in TrustedRoot for potential console output later.

Reply:
Yeah, I was thinking `trusted_root.json`.